src/Steps/Step.php

```diff
@@ -114,7 +120,13 @@ public function runCommand(OutputInterface $output, $command, $error = null, $ex
 
         // Run it
         $process->setTimeout(null);
+
+        $verbosity = $output->getVerbosity();
+        if ($allowDebugVerbosity && $verbosity === OutputInterface::VERBOSITY_DEBUG) {
+            $output->setVerbosity(OutputInterface::VERBOSITY_VERY_VERBOSE);
+        }
```

Comment:
```suggestion
        if (!$allowDebugVerbosity && $verbosity === OutputInterface::VERBOSITY_DEBUG) {
            $output->setVerbosity(OutputInterface::VERBOSITY_VERY_VERBOSE);
        }
```
I think this was the intention, right? i.e. we only change the verbosity if we _don't_ allow debug verbosity.

Reply:
Updated